platform/source/mbed_sdk_boot.c

```diff
@@ -76,6 +76,7 @@ int $Super$$main(void);
 int $Sub$$main(void)
 {
     mbed_main();
+    mbed_error_initialize();
```

Comment:
How about mbed_toolchain_init()? Isn't that missing as well? I think we talked about this already - what was the outcome?

Reply:
We thought to reach martin to get clarification but still yet to reach martin.

Reply:
Have you reached him and he hasnt replied yet or have you not reached him yet?

Reply:
I reached him today, still waiting for his response

Reply:
Russ was involved with most of these changes. Worth asking him on slack

Reply:
Ticket created to investigate this point.